llvm/test/CodeGen/EVM/commutable-ops.ll

```diff
@@ -0,0 +1,39 @@
+; RUN: llc < %s | FileCheck %s
```

Comment:
Perhaps worth merging with my test.
Also, it worth cheking that addmod, mulmod work as expected

Reply:
Done.